core/sets.gms

```diff
@@ -1537,18 +1532,16 @@ teBioPebiolc(all_te)      "biomass technologies using pebiolc"
 teNoTransform(all_te) "all technologies that do not transform energy but still have investment and O&M costs (like storage or grid)"
 /
        storspv       "storage technology for photo voltaic (PV)"
-       storwind      "storage technology for wind onshore"
-$IFTHEN.WindOff %cm_wind_offshore% == "1"
+***       storwind      "storage technology for wind onshore"
+       storwindon    "storage technology for wind onshore"
        storwindoff   "storage technology for wind offshore"
-$ENDIF.WindOff
        storcsp       "storage technology for concentrating solar power (CSP)"
 
        gridspv       "grid between areas with high pv production and the rest"
```

Comment:
As you are cleaning the code so thoroughly: 
could you remove all the different gridXXX techs and just replace them by one "VREgrid" tech? (it should take the "gridwind" parameters, as that was the only tech still used in the model)